scripts/test/test_prepare_ontology.py

```diff
@@ -0,0 +1,46 @@
+"""
+Tests for the prepare_ontology
+
+These tests run within the re_api docker image.
+"""
+import unittest
+import os
+from scripts.prepare_ontology import (
+    prepare_collections_file,
+    prepare_data_sources_file,
+    parse_input,
+    parse_namespace,
+    clean_up_data,
+)
+
+_TEST_DIR = "/app/scripts/test"
+_TEST_NAMESPACE = "fake_ontology"
+
+
+class Test_prepare_ontology(unittest.TestCase):
```

Comment:
can you add in some non-happy-path tests to ensure that any errors are caught and handled correctly by the code?